### ISSUE TITLE
[Bug] Invalid timer handle during timer end

### ISSUE THREAD

**Issue by MAGNAT2645** [Mentrillum/Slender-Fortress-Modified-Versions#20]
Sometimes SM throws this error:
```
L 01/15/2021 - 09:53:47: [SM] Plugin "disabled/sf2.smx" encountered error 23: Native detected error
L 01/15/2021 - 09:53:47: [SM] Invalid timer handle 5cc90bae (error 1) during timer end, displayed function is timer callback, not the stack trace
L 01/15/2021 - 09:53:47: [SM] Unable to call function "Timer_StartBossPackVote" due to above error(s).
```

**Comment by MAGNAT2645** [comment]
Sadly, this isn't solved, error from yesterday ([v1.6.1](https://github.com/Mentrillum/Slender-Fortress-Modified-Versions/releases/tag/v1.6.1mod)):
```
L 07/07/2021 - 17:56:10: [SM] Plugin "disabled/sf2.smx" encountered error 23: Native detected error
L 07/07/2021 - 17:56:10: [SM] Invalid timer handle bebd04e7 (error 1) during timer end, displayed function is timer callback, not the stack trace
L 07/07/2021 - 17:56:10: [SM] Unable to call function "Timer_StartBossPackVote" due to above error(s).
```

**Comment by Mentrillum** [comment]
Most likely fixed in 1.7.0, unconfirmed.
